mydiscord.py

```diff
@@ -141,8 +139,6 @@ async def on_message(message):
                 textFile.writelines(message.content)
                 textFile.close()
 
-                # --- parse the incoming Discord message
```

Comment:
This is what was causing the conflict. parsemessage() was being called twice. Once the worshipschedule.txt file was updated with the correct song case, parsemessage() was called again overwriting it with the original message.